spec.html

```diff
@@ -42956,12 +43435,32 @@ <h1>SharedArrayBuffer.prototype [ @@toStringTag ]</h1>
 
     <emu-clause id="sec-properties-of-the-sharedarraybuffer-instances">
       <h1>Properties of SharedArrayBuffer Instances</h1>
-      <p>SharedArrayBuffer instances inherit properties from the SharedArrayBuffer prototype object. SharedArrayBuffer instances each have an [[ArrayBufferData]] internal slot and an [[ArrayBufferByteLength]] internal slot.</p>
+      <p>SharedArrayBuffer instances inherit properties from the SharedArrayBuffer prototype object. SharedArrayBuffer instances each have an [[ArrayBufferData]] internal slot. SharedArrayBuffer instances which are not growable each have an [[ArrayBufferByteLength]] internal slot. SharedArrayBuffer instances which are growable each have an [[ArrayBufferByteLengthData]] internal slot and an [[ArrayBufferMaxByteLength]] internal slot.</p>
 
       <emu-note>
         <p>SharedArrayBuffer instances, unlike ArrayBuffer instances, are never detached.</p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-growable-sharedarraybuffer-guidelines">
+      <h1>Growable SharedArrayBuffer Guidelines</h1>
```

Comment:
I think there should be discussion/mention of growable SharedArrayBuffers at the top of 25.2, and maybe this whole section.

Reply:
Done.